var/spack/repos/builtin/packages/plumed/package.py

```diff
@@ -41,14 +41,24 @@ class Plumed(Package):
     and C/C++ codes.
     """
     homepage = 'http://www.plumed.org/'
-    url = 'https://github.com/plumed/plumed2/archive/v2.2.3.tar.gz'
+    url = 'https://github.com/plumed/plumed2'
 
-    version('2.2.3', 'a6e3863e40aac07eb8cf739cbd14ecf8')
+    version('2.2.3', git="https://github.com/plumed/plumed2.git", tag='v2.2.3')
```

Comment:
If policies didn't change lately I think pointing to a tag name is not the preferred way to go for retrieving a package (because of security issues)  @tgamblin ?